### PR TITLE
Create Transactions ViewModel

### PR DESCRIPTION
**Pull request by Jeizard**

### CODE REVIEW

File: app/src/main/java/com/jeizard/mbanking/ui/screens/common/view_models/TransactionsViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import com.jeizard.mbanking.models.Transaction
+import com.jeizard.mbanking.models.TransactionStatus
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+
+class TransactionsViewModel : ViewModel() {
+    private val _transactions = MutableStateFlow<List<Transaction>>(emptyList())
+    val transactions: StateFlow<List<Transaction>> get() = _transactions
+
```
Comment (Razor0744) [inline app/src/main/java/com/jeizard/mbanking/ui/screens/common/view_models/TransactionsViewModel.kt]: val transactions: StateFlow<> = _transactions.asStateFlow

File: app/src/main/java/com/jeizard/mbanking/ui/screens/common/view_models/TransactionsViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                Transaction(
+                    company = "OOO \"Company\"",
+                    number = "f4345jfshjek3450",
+                    date = "29.05.2024",
+                    amount = "$10.09",
+                    status = TransactionStatus.Executed
+                )
+            )
+            _transactions.value = sampleTransactions
+        }
```
Comment (Razor0744) [inline app/src/main/java/com/jeizard/mbanking/ui/screens/common/view_models/TransactionsViewModel.kt:108]: use _transactions.emit() with coroutine if you can

File: app/src/main/java/com/jeizard/mbanking/ui/screens/common/view_models/TransactionsViewModel.kt (hunk 1 above)
Comment (Jeizard) [reply app/src/main/java/com/jeizard/mbanking/ui/screens/common/view_models/TransactionsViewModel.kt]: I did it

File: app/src/main/java/com/jeizard/mbanking/ui/screens/common/view_models/TransactionsViewModel.kt (hunk 2 above)
Comment (Jeizard) [reply app/src/main/java/com/jeizard/mbanking/ui/screens/common/view_models/TransactionsViewModel.kt:108]: I did it

**Pull request merged**